examples/connectivity/plot_connectivity_measures.py

```diff
@@ -0,0 +1,95 @@
+"""
+Comparing different functional connectivity measures
+====================================================
+
+This example compares different measures of functional connectivity between
+regions of interest : correlation, partial correlation, as well as a measure
+called tangent. The resulting connectivity coefficients are used to
+classify ADHD vs control subjects and the tangent measure outperforms the
+standard measures.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
+
+# Extract regions time series signals
+import nilearn.input_data
+masker = nilearn.input_data.NiftiMapsMasker(
+    atlas.maps, resampling_target="maps", detrend=True,
+    low_pass=None, high_pass=None, t_r=2.5, standardize=False,
+    memory='nilearn_cache', memory_level=1)
+subjects = []
+sites = []
+adhds = []
+for func_file, phenotypic in zip(dataset.func, dataset.phenotypic):
+    # keep only 3 sites, to save computation time
+    if phenotypic['site'] in [b'"NYU"', b'"OHSU"', b'"NeuroImage"']:
+        time_series = masker.fit_transform(func_file)
+        subjects.append(time_series)
+        sites.append(phenotypic['site'])
+        adhds.append(phenotypic['adhd'])  # ADHD/control label
+
+# Estimate connectivity
+import nilearn.connectome
+kinds = ['tangent', 'partial correlation', 'correlation']
+individual_connectivity_matrices = {}
+mean_connectivity_matrix = {}
+for kind in kinds:
+    conn_measure = nilearn.connectome.ConnectivityMeasure(kind=kind)
+    individual_connectivity_matrices[kind] = conn_measure.fit_transform(
```

Comment:
People using this code to compare conditions/groups may fit separately the individual time-series for each condition/group. This will not be meaningful for the tangent measure. Shall comment about this ?